packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -88,15 +88,7 @@ const notifySlack = async (
   const {botAccessToken, userId} = auth
   const manager = new SlackServerManager(botAccessToken!)
   const res = await manager.postMessage(channelId!, slackMessage, notificationText, ts)
-  segmentIo.track({
-    userId,
-    event: 'Slack notification sent',
-    properties: {
-      teamId,
-      notificationEvent: event,
-      ...segmentProperties
-    }
-  })
+  analytics.slackNotificationSent(userId, teamId, event, segmentProperties)
```

Comment:
-1 let's type `segmentProperties` properly while we're at it: `{reflectionGroupId?: string} | undefined`